lib/ApplicationFeatures/ApplicationServer.cpp

```diff
@@ -885,3 +926,36 @@ std::string_view ApplicationServer::stringifyState(State state) {
   TRI_ASSERT(false);
   return "unknown";
 }
+
+void ApplicationServer::recordAPICall(arangodb::rest::RequestType requestType,
+                                      std::string_view path,
+                                      std::string_view database) {
+  if (_recordApiCalls && _apiCallRecord != nullptr) {
+    _apiCallRecord->prepend(ApiCallRecord(requestType, path, database));
+  }
+}
+
+void ApplicationServer::cleanupLoop() {
+  while (!_stopCleanupThread.load(std::memory_order_relaxed)) {
+    // Get the trash and measure the time
+    auto start = std::chrono::steady_clock::now();
+    auto trash = _apiCallRecord->getTrash();
```

Comment:
Instead of having to work with this return type, I t think it would be much easier if we have just a method
```
auto count = _apiCallRecord->trash()
```
which clears itself and returns the number of items trashed.

Reply:
Done.